examples/clock-example/elements/hand.js

```diff
@@ -0,0 +1,36 @@
+const Tram = require('../../../tram-one')
+
+// the hands to render in the clock
+const svg = Tram.svg()
+
+const handStyle = (degrees) => `
+  transform-origin: center;
+  transform: rotate(${degrees}deg);
+`
+
+const degreesFromAttrs = (attrs) => {
+  return 360 * (attrs['value'] / attrs['max-value'])
+}
+
+const halfHand = svg`
```

Comment:
could you label this as halfHand definition? this might be a tad confusing for people that don't use svgs a lot.
Or maybe you could move this to an area where all the defs can live.

Reply:
I've been thinking back and forth about moving this definition out, and potentially just creating one instance of the def in `clock.js`, but then this element becomes dependent on an outside dependency, which feels really weird.

I can rename it to be something more meaningful.